configs/warp-route-chain-config.json

```diff
@@ -1,6 +1,6 @@
 {
   "goerli": {
-    "id": 5,
+    "domainId": 5,
```

Comment:
The multiprovider uses id as domainId. This field should only be necessary when id isn't set, but then you should have both. Do you happen to know why this was needed? There may be a bug in the MP or deploy code

Reply:
IIRC it was this line https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/31a102a0a08227ab862178e27a73e2feb393ef81/typescript/sdk/src/deploy/router/HyperlaneRouterDeployer.ts#L87

which will hit this line https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/31a102a0a08227ab862178e27a73e2feb393ef81/typescript/sdk/src/providers/MultiProvider.ts#L152

Which does not use the ID?

Reply:
Sorry i wasn't clear. I'm saying this field should be called chainId, not domainId, because domainId is only needed if it !== chainId (per that MP line you linked to).

Reply:
ahh yes, i'll change